root/release/edit/tracklist.tt

```diff
@@ -66,6 +66,22 @@
         </label>
       </td>
     </tr>
+    <tr>
+    <td>
+      <label>
+      <input type="checkbox" name="use-custom-delimiter" data-bind="checked: useCustomDelimiter" />
+        [%= l('Use custom artist delimeter') %]
```

Comment:
s/delimeter/delimiter/

Reply:
What do you mean?

Reply:
Delimiter is spelled wrong (sorry for being oblique, s// performs a substitution in vim or perl)

Reply:
Oh right 👍👍